assets/sass/components/tour-tooltip/_googlesitekit-tour-tooltip.scss

```
@@ -220,3 +220,65 @@
 	margin-top: 42px !important;
 	top: 0 !important;
 }
+
+// Setup Flow Refresh variant styles.
+.googlesitekit-tour-tooltip--setupFlowRefresh {
+
+	.googlesitekit-tooltip-title {
+		font-family: $f-primary;
+		font-size: $fs-title-lg;
+		font-weight: $fw-medium;
+		letter-spacing: $ls-xxs;
+		line-height: $lh-title-lg;
+		margin-bottom: 8px;
+	}
```

According to the Figma designs, typography for this should be `title-md`.


```suggestion
	.googlesitekit-tooltip-title {
		font-family: $f-primary;
		font-size: $fs-title-md;
		font-weight: $fw-medium;
		letter-spacing: $ls-xxs;
		line-height: $lh-title-md;
		margin-bottom: 8px;
	}
```